crates/node/src/types.rs

```diff
@@ -30,10 +30,12 @@ pub enum StreamMessage<'a> {
         party_id: PublicKey,
         // nonce: usize,
```

Comment:
this was here already

also, let's use the `next_` prefix so it's easier to follow